### ISSUE TITLE
Flutter can't detect Android CommandLine Tools

### ISSUE THREAD

**Issue by Rafeqm** [ScoopInstaller/Extras#8172]
## Bug Report

**Package Name:** `android-sdk`

### Current Behaviour

Whenever I run `flutter doctor` it gives this warning:
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.10.3, on Microsoft Windows [Version 10.0.22000.527], locale en-US)
[✗] Android toolchain - develop for Android devices
    ✗ cmdline-tools component is missing
      Run `path/to/sdkmanager --install "cmdline-tools;latest"`
      See https://developer.android.com/studio/command-line for more details.
[✗] Chrome - develop for the web (Cannot find Chrome executable at .\Google\Chrome\Application\chrome.exe)
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.
[✗] Visual Studio - develop for Windows
    ✗ Visual Studio not installed; this is necessary for Windows development.
      Download at https://visualstudio.microsoft.com/downloads/.
      Please install the "Desktop development with C++" workload, including all of its default components
[✓] Android Studio (version 2021.1)
[✓] Connected device (2 available)
[✓] HTTP Host Availability

! Doctor found issues in 3 categories.
```

So only the first warning that get me nerve. I thought I can solve it after installing `android-clt`. But still it is not solved because `android-clt` is installed in different directory than `%USERPROFILE%\scoop\apps\android-sdk\current\cmdline-tools\latest\bin`. Flutter cannot detect it since that folder is not linked to `%USERPROFILE%\scoop\apps\android-clt\current\cmdline-tools\bin`.

### Expected Behaviour

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.10.3, on Microsoft Windows [Version 10.0.22000.527], locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 32.1.0-rc1)
[✗] Chrome - develop for the web (Cannot find Chrome executable at .\Google\Chrome\Application\chrome.exe)
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.
[✗] Visual Studio - develop for Windows
    ✗ Visual Studio not installed; this is necessary for Windows development.
      Download at https://visualstudio.microsoft.com/downloads/.
      Please install the "Desktop development with C++" workload, including all of its default components
[✓] Android Studio (version 2021.1)
[✓] Connected device (2 available)
[✓] HTTP Host Availability

! Doctor found issues in 2 categories.
```

### Additional context/output

Maybe `android-clt` in the main bucket should be merge to this manifest? And then deprecate `android-clt`.

### Possible Solution

I worked around it and it is actually `cmdline-tools` folder is not persisted which means my Android CommandLine Tools is still on previous root folder of `android-sdk`. therefore,  this `cmdline-tools` folder should be persisted.

### System details

**Windows version:** 11

**OS architecture:** 64bit

**PowerShell version:** v7.2.2

**Comment by Rafeqm** [comment]
I'm currently working on a PR on this and trying to find more similar issue related with Android development and Flutter

**Comment by tetsuhou** [comment]
If using `android-clt`, then according to https://stackoverflow.com/questions/65262340/cmdline-tools-could-not-determine-sdk-root, this can be solved by moving the folders or files from `\SOMEPATH\cmdline-tools` to `\SOMEPATH\cmdline-tools\latest`, after that there is no need to "launch sdkmanager with --sdk_root=$env:ANDROID_SDK_ROOT", and it should also make flutter work.

**Comment by Rafeqm** [comment]
@tetsuhou This has been solved in #8252

**Comment by tetsuhou** [comment]
@Rafeqm Thank you for your reply, the solution I mentioned is the case of not using android-sdk but only android-clt.
